filebeat/module/mariadb/error/ingest/pipeline.json

```diff
@@ -0,0 +1,50 @@
+{
+  "description": "Pipeline for parsing MariaDB error logs. It currently uses message1 and message2 to workaround bug https://github.com/elastic/elasticsearch/issues/22117",
+  "processors": [{
```

Comment:
Can we use here directly ECS fields?

Reply:
I'd try to keep it as similar to mysql as possible in case we merge them.